ep/build-velox/src/get_velox.sh

```diff
@@ -2,8 +2,11 @@
 
 set -exu
 
-VELOX_REPO=https://github.com/oap-project/velox.git
-VELOX_BRANCH=main
+#VELOX_REPO=https://github.com/oap-project/velox.git
+#VELOX_BRANCH=main
+
+VELOX_REPO=https://github.com/JkSelf/velox.git
+VELOX_BRANCH=parquet_write_use_temp
```

Comment:
just merged the commit in velox, can you please do a rebase?

Reply:
Done.